knowrob_common/knowrob_common/src/main/java/org/knowrob/prolog/PrologInterface.java

```diff
@@ -140,7 +141,7 @@ public static void initLocalProlog(String initFile) {
     		// Build the result
     		for (int i=0; i<solutions.length; i++) {
     			@SuppressWarnings("rawtypes")
-				Hashtable solution = solutions[i];
+				Map<String, Term> solution = solutions[i];
```

Comment:
is the SuppressWarnings still needed?

Reply:
Should be okay to remove it, since Map is typed.